sdcm/nemesis.py

```diff
@@ -2314,7 +2318,7 @@ def disrupt_grow_shrink_cluster(self):
             if self._is_it_on_kubernetes():
                 self.set_last_node_as_target()
             else:
-                self.set_target_node()
+                self.set_target_node(dc_idx=self.target_node.dc_idx)
```

Comment:
maybe this would be clearer ?
```suggestion
                self.set_target_node(dc_idx=new_node.dc_idx)
```

Reply:
Yes, thanks.

Reply:
Alex, please fix according to Israel's comment